### ISSUE TITLE
Blog colours (mostly) do not pass contract checker for accessibility

### ISSUE THREAD

**Issue by chriswilty** [ScottLogic/blog#248]
## Problem

There are four colour schemes for our blog posts: teal, pink, orange, yellow - see `scss/blog-colour-schemes.scss`.

Of these, only the pink colour scheme passes contrast checks at AA standard, on a white background:

![image](https://github.com/user-attachments/assets/60d6053e-f6d2-4fe0-b022-bc2b19ae442e)

The others fail, with yellow being the worst offender:

![image](https://github.com/user-attachments/assets/e5d81db6-58f7-40f5-b946-e8e3bcf13ad3)

It's important to note this indicates a general problem with our branding, though the plain white background is clearly not helping, as with a dark background the remaining three pass, e.g.

![image](https://github.com/user-attachments/assets/a776288e-9b17-4fee-a50f-a5dd81c5cf56)

## Solution?

For our blog, we might want to consider straying beyond our strict corporate theme colours, even if it just means darkening them a little, e.g. 

`#EC6F27`
![image](https://github.com/user-attachments/assets/1db3ba6a-673f-4c92-a894-d513ada4356f)

`#27A4AA`
![image](https://github.com/user-attachments/assets/2f32f920-0b6e-4403-9518-70b5cdadeef4)

Yellow is never going to pass though, on a white background  🫥 

For this issue, I suggest adopting the above darkened colours and removing yellow entirely.
